docs/develop/manifest-element-ordering.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to find the proper order of manifest elements
+description: Learn how to find the correct order in which to place child elements in a parent element.
+ms.date: 11/16/2018
+---
+
+# How to find the proper order of manifest elements
+
+The XML elements in the manifest of an Office Add-in must not only be under to the proper parent element; they must be in a specific order, relative to each other, under the parent. For example, In the `<OfficeApp>` element, the `<Id>`, `<Version>`, `<ProviderName>` must appear in that order. If an `<AlternateId>` element is added, it must be between the `<ID>` and `<Version>` element. If any element is in the wrong order, your manifest will not be valid and your add-in will not load.
+
+> [!NOTE]
+> The [Office Add-in Validator](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/troubleshoot-manifest#validate-your-manifest-with-the-office-add-in-validator) uses the same error message when an element is out-of-order as it does when an element is under the wrong parent. The error will say that the child element is is not a valid child of the parent element. If you get such an error but the reference documentation for the child element indicates that it *is* valid for the parent, then the problem is likely that the child has been placed in the wrong order.
+
+The required ordering is specified in the XSD files in the [Schemas](https://github.com/OfficeDev/office-js-docs-pr/tree/docs/overview/schemas) folder. The XSD files are categorized into subfolders for taskpane, content, and mail add-ins.
```

Comment:
I'd probably lead with this, instead of the example. That way, there's a point of reference as to why AlternativeId goes between Id and Version before that claim is made.